kolibri/plugins/facility/assets/src/routes.js

```diff
@@ -25,20 +26,28 @@ import {
 import { PageNames } from './constants';
 
 function facilityParamRequiredGuard(toRoute, subtopicName) {
+  const { isNavigationFailure, NavigationFailureType } = VueRouter;
   if (store.getters.userIsMultiFacilityAdmin && !toRoute.params.facility_id) {
-    router.replace({
-      name: 'ALL_FACILITIES_PAGE',
-      params: { subtopicName },
-    });
-    return true;
+    router
+      .replace({
+        name: 'ALL_FACILITIES_PAGE',
+        query: { subtopicName },
+      })
+      .catch(e => {
+        if (!isNavigationFailure(e, NavigationFailureType.duplicated)) {
+          console.debug(e);
+          throw Error(e);
+        }
+      });
   }
+  return true;
```

Comment:
This is now always returning `true` from this guard - regardless of whether the condition pertains or not - this should be inside the if statement.